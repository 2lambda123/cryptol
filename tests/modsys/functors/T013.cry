
submodule A where
  x = 2

submodule F where
  import signature submodule A
