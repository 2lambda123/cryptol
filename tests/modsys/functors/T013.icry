:load T013.cry
