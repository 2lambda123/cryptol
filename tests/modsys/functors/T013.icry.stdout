Loading module Cryptol
Loading module Cryptol
Loading module Main

[error] at T013.cry:6:30--6:31
    • Expected a signatre
    • `submodule Main::A` is a module
